cmd/clusterctl/cmd/move.go

```diff
@@ -80,6 +85,8 @@ func init() {
 		"Write Cluster API objects and all dependencies from a management cluster to directory.")
 	moveCmd.Flags().StringVar(&mo.fromDirectory, "from-directory", "",
 		"Read Cluster API objects and all dependencies from a directory into a management cluster.")
+	moveCmd.Flags().BoolVar(&mo.hideAPIWarnings, "hide-api-warnings", true,
```

Comment:
Wondering if we should have a slightly different name (or just usage text), as we don't hide all warnings

Reply:
We have some set of warnings we want to hide. That set may change over time.

And user preferences could change over time, too.

How about we use an argument?

- `all`: hide all warnings.
- `default`: the currently chosen set of warnings to hide.
- `finalizername`, `foo`, `bar,`: names of specific warnings to hide.

We don't need to support all these arguments to start. We could start simple, with just `default` and `all`:

```
--hide-api-warnings=all
```

```
--hide-api-warnings=default
```

And we can `default` the default value, so that the above is equivalent to the flag with no argument, e.g.

```
--hide-api-warnings
```

Later, if users ask to define their own set of warnings to hide, we can support that, e.g.:

```
---hide-api-warnings=foo,bar
```

Reply:
Sounds good to me